src/components/NewUserForm/NewUserForm.js

```diff
@@ -31,13 +31,13 @@ function NewUserForm() {
     if (password === passwordConfirmation && password.length >= 8) {
       postNewUser(user)
       .then(data => loginCheck(data))
+      .catch(err => console.log(err))
     } else {
       return 'Password need to be 8 letters long.'
```

Comment:
Maybe change this to characters instead of letters? Unless we're only allowing letters.

Reply:
fixed